### ISSUE TITLE
Question about the training data for the challenge

### ISSUE THREAD

**Issue by woxihuanjiangguo** [autonomousvision/navsim#92]
Hello authors, 

Are the navtrain/trainval data splits the only available training data for the challenge? To confirm, we should not use navhard/navtest/test splits for training, correct?

Thanks!

**Comment by kashyap7x** [comment]
Hello,

Thanks for your question! You are correct, **using the navhard/navtest/test/warmup/private_test splits for training is not allowed**. Using any other publicly available datasets or pretrained weights is allowed. Furthermore, to be eligible for awards, the use of data must be described explicitly in the technical report for your submission.

We will update our documentation to emphasize this point.

Best,
Kashyap

**Comment by woxihuanjiangguo** [comment]
Thanks for your clarification!
